### PR TITLE
Improve unit test coverage for network policy validation.

### PR DESCRIPTION
**Pull request by xiangpengzhao**
**What this PR does / why we need it**:
`ok  	k8s.io/kubernetes/pkg/apis/networking/validation	0.074s	coverage: 87.8% of statements`

to 

`ok  	k8s.io/kubernetes/pkg/apis/networking/validation	0.082s	coverage: 100.0% of statements`

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/networking/validation/validation_test.go [hunk 1]
```diff
@@ -246,6 +276,38 @@ func TestValidateNetworkPolicy(t *testing.T) {
...
+								NamespaceSelector: &metav1.LabelSelector{
+									MatchLabels: map[string]string{"c": "d"},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+		"no from and to type": {
```
Comment (cmluciano) [inline pkg/apis/networking/validation/validation_test.go]: Can you change this to 

`missing from and to type`

**Review by cmluciano — CHANGES_REQUESTED** [review]
Just one comment, but otherwise this looks good

File: pkg/apis/networking/validation/validation_test.go (hunk 1 above)
Comment (xiangpengzhao) [reply pkg/apis/networking/validation/validation_test.go]: Thanks! Done.

**Comment by xiangpengzhao** [comment]
@cmluciano updated. PTAL. Thanks!

**Review by cmluciano — APPROVED** [review]

**Comment by thockin** [comment]
/lgtm
/approve

Would be nice is someone went through and added comments to each test case about why it is interesting.

**Pull request merged**